element_data_loader/Suite2p_trigger_individual/Suite2p_trigger.py

```diff
@@ -0,0 +1,180 @@
+from suite2p import run_s2p
+from suite2p.extraction import dcnv
+import numpy as np
+import os
+import warnings
+
+
+def motion_correction_suite2p(ops, db):
+    """Performs registration using Suite2p package, 
+    Make sure the 'do_registration' key is set to 1 to perform registration
+
+    Args:
+        ops (dict): ops dictionary can be obtained by using default_ops() function. 
+                    It contains all options and default values used to perform 
+                    preprocessing.
+        db (dict): dictionary that includes paths pointing towards the input data,
+                   and where to store the data
+
+    Returns:
+        dict: Returns a dictionary with x and y shifts
+        data.bin: Creates and saves a binary file on your local path. If 
+                  delete_bin is set to True, the binary file is deleted after
+                  processing.
+        ops.npy: Creates and saves the options dictionary in a numpy array file 
+                 in the specified path. The ops file gets updated during the 
+                 cell detection and the deconvultion steps.
+    """
+
+    # Nonrigid motion correction
+
+    if ops['nonrigid']:
+
+        print("==========RUNNING NON-RIGID REGISTRATION==========")
+        warnings.warn('You are running registration/ motion correction using \
+                      Suite2p module, this requires do_registration=1,\
+                      roidetect=False, spikedetect=False. If the ops dictionary\
+                      has values differing from these, the flags will \
+                      automatically be se set to it.')
+
+        ops.update(do_registration = 1, 
+                    roidetect = False,
+                    spikedetect = False,
+                    delete_bin = True)
```

Comment:
Please refactor the if-else statement to move common blocks of code out of the statement (e.g. warning statement, and ops update).